### DISCUSSION TITLE
What if we add the ability to connect to sites with a specific domain directly?

### DISCUSSION THREAD

**Discussion by QuMix64** [dragonfruitnetwork/onionfruit#52] [Ideas]
Hello, I think it would be cool to add the ability to not use tor with a specific domain. For example, I want to connect to .ru sites directly, and to all others through tor. Is it possible to implement this?

**Comment by aspriddell** [comment]
This is something I 100% would like to see but would require using something like Windows Filtering Platform (WFP) to filter out requests based on app or domain, which may end up in kernel-driver territory (which I'd like to avoid if possible).

There is also a way to map certain domains to custom IP addresses/exit nodes using `MapAddress`, which is what #20 would have used but never got around to implementing it which would be more achievable.
